next.config.js

```diff
@@ -1,16 +1,40 @@
 /** @type {import('next').NextConfig} */
+require('webpack')
+require('path')
 
 // eslint-disable-next-line @typescript-eslint/no-var-requires
 const { withPlausibleProxy } = require('next-plausible')
 
 module.exports = withPlausibleProxy()({
-  reactStrictMode: true,
+  reactStrictMode: false,
   serverRuntimeConfig: {
     APP_ROOT: __dirname,
   },
+  compiler: {
+    styledComponents: false,
+  },
+  typescript: {
+    ignoreBuildErrors: true,
+  },
   webpack: (config, options) => {
     const { dir, defaultLoaders } = options
 
+    config.resolve.alias['path'] = require.resolve('path-browserify')
+
+    // see: https://github.com/solidity-parser/parser/issues/47
+    config.resolve.alias['@solidity-parser/parser'] =
+      '@solidity-parser/parser/dist/index.iife.js'
+
+    config.module.rules.push({
+      // We tell webpack to append "module.exports = SolidityParser;" at the end of the file.
+      test: require.resolve('@solidity-parser/parser/dist/index.iife.js'),
+      loader: 'exports-loader',
+      options: {
+        type: 'commonjs',
+        exports: 'single SolidityParser',
+      },
+    })
```

Comment:
Looking at the referenced issue (which is closed now), it seems it was fixed in latest release: https://github.com/solidity-parser/parser/blob/master/CHANGELOG.md